deepspeed/module_inject/replace_policy.py

```diff
@@ -405,8 +405,8 @@ class MegatronLayerPolicy(TransformerPolicy):
     version = 0
     moe_type = 'standard'
 
-    def __init__(self, client_module, inference=True):
-        super().__init__(inference)
+    def __init__(self, client_module, inference=True, megatron_v2=True):
+        super().__init__(inference, megatron_v2=megatron_v2)
```

Comment:
Let's maybe get it from the MegatronLayerPolicy static argument, like this: `MegatronLayerPolicy=MegatronLayerPolicy.megatron_v2`

Reply:
Updated in latest commit, thanks for the feedback